modules/installation-approve-csrs.adoc

```diff
@@ -19,20 +19,34 @@ signing requests (CSRs) for the machines that you added.
 .Procedure
 
 . Review the pending certificate signing requests (CSRs) and ensure that the
-requests are for the machines that you added to the cluster:
+you see a client and server request with `Pending` status for each machine that
+you added to the cluster:
 +
 ----
 $ oc get csr
 
 NAME        AGE     REQUESTOR                                                                   CONDITION
-csr-8b2br   15m     system:serviceaccount:openshift-machine-config-operator:node-bootstrapper   Approved,Issued
-csr-8vnps   15m     system:serviceaccount:openshift-machine-config-operator:node-bootstrapper   Approved,Issued
-csr-b96j4   25s     system:node:ip-10-0-52-215.us-east-2.compute.internal                       Approved,Issued
-csr-bfd72   5m26s   system:node:ip-10-0-50-126.us-east-2.compute.internal                       Pending
+csr-8b2br   15m     system:serviceaccount:openshift-machine-config-operator:node-bootstrapper   Pending <1>
+csr-8vnps   15m     system:serviceaccount:openshift-machine-config-operator:node-bootstrapper   Pending
+csr-bfd72   5m26s   system:node:ip-10-0-50-126.us-east-2.compute.internal                       Pending <2>
 csr-c57lv   5m26s   system:node:ip-10-0-95-157.us-east-2.compute.internal                       Pending
 ----
+<1> A client request CSR.
+<2> A server request CSR.
++
+In this example, two machines are joining the cluster.
+
+. After all of the pending CSRs for the machines you added are in `Pending`
+status, approve the CSRs for your cluster machines:
++
+[NOTE]
+====
+Because the CSRs rotate automatically, approve your CSRs within half an hour
```

Comment:
cc @sjenning @rphillips

Reply:
When trying to get a certificate (client/server), kubelet (node) creates a CSR and waits for it to be Approved and Issued. If the CSR is not approved for _some time_ (Pod team can provide more guidance on what this period is), kubelet (node) creates a new request for certificate with new CSR.

So there's a chance that there might be more CSR objects in Pending for the same purpose.

I'm hoping Pod Team (cc @sjenning ) can provide information on the exact behavior.

Reply:
the _some time_ the kubelet will wait for a certificate to be issues is 1 hr.

Reply:
Thanks! I've updated the note to say that it takes an hour for the certs to rotate and that if you wait longer than that, you might have to approve more certs.